oracle/README.md

```diff
@@ -37,31 +37,57 @@ oracle/
 
 TODO: need UI for this to be relevant
 
-### Node
-In order to run the Oracle node make sure you are in the root of this project i.e `/path/to/oracle/<you are here>`
+The project can be started by executing the following steps:
 
-Copy the `.env.example` file into a new file called `.env`.  Then insert your api key into the `API_URL`
+1. Change into the `Oracle` directory
 
-**_Note:_** You do not need an api key to run any tests
+    ```bash
+    cd <path>/sway-applications/oracle/<you are here>
+    ```
 
+2. Copy and paste the `.env.example` file into a new file called `.env`
 
-Then start a local fuel-core instance
-```bash
-fuel-core run --chain packages/node/.chainConfig.json
-```
+    ```bash
+    cp packages/node/.env.example packages/node/.env
+    ```
 
-Then deploy an instance of the Oracle contract
-```bash
-forc deploy --path packages/contract --url localhost:4000 --unsigned
-```
+### Environment Variables
 
-Finally start the Oracle node
-```bash
-cargo run
-```
+| Name               | Description |
+|--------------------|-------------|
+| API_URL            | The URL the node uses to fetch the latest price for the asset tracked by the oracle. |
+| ORACLE_CONTRACT_ID | Deterministic contract id of the oracle contract which is deployed in a later step |
+| WALLET_SECRET      | Private key of the first deterministic wallet provided by the fuels-rs sdk.  It is also the private key corresponding to the `owner` address specified in the oracle contract's `Forc.toml`.  It is configured to have the maximum amount of the `BASE_ASSET` by the local `fuel-core` instance spun up in the step 4. |
+| FUEL_PROVIDER_URL  | Fuel-core network url normally set as http://localhost:4000/graphql for development |
+
+3. The oracle node relies on an external 3rd-party service to get price information to provide to the oracle contract.  We do not endorse this service neither are we affiliated with them in any way.  We only use the service for demonstration purposes.  If you wish to run the node you can sign-up for a free api key [here](https://www.cryptocompare.com/).  In the newly copied `.env` file there is a variable `API_URL` which ends with `<your api key here>`.  This section should be replaced with your API key.  If you wish to use another pricing api service feel free to replace `API_URL` entirely.
```

Comment:
This is 1 large block of text which mixes commentary and instructions. This should be separated so that the instructions are instructions and any relevant commentary is referenced elsewhere / cut down to be concise.